File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Invs/INV6002U00GrdINV6002Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Invs
{
    public class INV6002U00GrdINV6002Info
    {
        private String _jaeryoCode;
        private String _jaeryoName;
        private String _existCount;
        private String _subulDanuiName;
        private String _inputDate;
        private String _inputUser;
        private String _jaryoGubun;
        private String _magamMonth;

        public String JaeryoCode
        {
            get { return this._jaeryoCode; }
            set { this._jaeryoCode = value; }
        }

        public String JaeryoName
        {
            get { return this._jaeryoName; }
            set { this._jaeryoName = value; }
        }

        public String ExistCount
        {
            get { return this._existCount; }
            set { this._existCount = value; }
        }

        public String SubulDanuiName
        {
            get { return this._subulDanuiName; }
            set { this._subulDanuiName = value; }
        }

        public String InputDate
        {
            get { return this._inputDate; }
            set { this._inputDate = value; }
        }

        public String InputUser
        {
            get { return this._inputUser; }
            set { this._inputUser = value; }
        }

        public String JaryoGubun
        {
            get { return this._jaryoGubun; }
            set { this._jaryoGubun = value; }
        }

        public String MagamMonth
        {
            get { return this._magamMonth; }
            set { this._magamMonth = value; }
        }

        public INV6002U00GrdINV6002Info() { }

        public INV6002U00GrdINV6002Info(String jaeryoCode, String jaeryoName, String existCount, String subulDanuiName, String inputDate, String inputUser, String jaryoGubun, String magamMonth)
        {
            this._jaeryoCode = jaeryoCode;
            this._jaeryoName = jaeryoName;
            this._existCount = existCount;
            this._subulDanuiName = subulDanuiName;
            this._inputDate = inputDate;
            this._inputUser = inputUser;
            this._jaryoGubun = jaryoGubun;
            this._magamMonth = magamMonth;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/invs/INV6002U00GrdINV6002Info.java
package nta.med.data.model.ihis.invs;

public class INV6002U00GrdINV6002Info {
	private String jaeryoCode          ;
	private String jaeryoName          ;
	private String existCount          ;
	private String subulDanuiName     ;
	private String inputDate           ;
	private String inputUser           ;
	private String jaryoGubun          ;
	private String magamMonth          ;
	public INV6002U00GrdINV6002Info(String jaeryoCode, String jaeryoName, String existCount, String subulDanuiName,
			String inputDate, String inputUser, String jaryoGubun, String magamMonth) {
		super();
		this.jaeryoCode = jaeryoCode;
		this.jaeryoName = jaeryoName;
		this.existCount = existCount;
		this.subulDanuiName = subulDanuiName;
		this.inputDate = inputDate;
		this.inputUser = inputUser;
		this.jaryoGubun = jaryoGubun;
		this.magamMonth = magamMonth;
	}
	public String getJaeryoCode() {
		return jaeryoCode;
	}
	public void setJaeryoCode(String jaeryoCode) {
		this.jaeryoCode = jaeryoCode;
	}
	public String getJaeryoName() {
		return jaeryoName;
	}
	public void setJaeryoName(String jaeryoName) {
		this.jaeryoName = jaeryoName;
	}
	public String getExistCount() {
		return existCount;
	}
	public void setExistCount(String existCount) {
		this.existCount = existCount;
	}
	public String getSubulDanuiName() {
		return subulDanuiName;
	}
	public void setSubulDanuiName(String subulDanuiName) {
		this.subulDanuiName = subulDanuiName;
	}
	public String getInputDate() {
		return inputDate;
	}
	public void setInputDate(String inputDate) {
		this.inputDate = inputDate;
	}
	public String getInputUser() {
		return inputUser;
	}
	public void setInputUser(String inputUser) {
		this.inputUser = inputUser;
	}
	public String getJaryoGubun() {
		return jaryoGubun;
	}
	public void setJaryoGubun(String jaryoGubun) {
		this.jaryoGubun = jaryoGubun;
	}
	public String getMagamMonth() {
		return magamMonth;
	}
	public void setMagamMonth(String magamMonth) {
		this.magamMonth = magamMonth;
	}
}
